contrib/kube-prometheus/jsonnetfile.lock.json

```diff
@@ -78,7 +78,7 @@
                     "subdir": "Documentation/etcd-mixin"
                 }
             },
-            "version": "ba606bf85edfb3007f27a97d41f54e3fe3f70ce6"
+            "version": "7a759c18d294698f537f8be91927354818a71e51"
         }
     ]
-}
+}
```

Comment:
@brancz are you running an old version of jb? I thought that new versions introduced a newline at the end of the lock file

Reply:
You're totally right, re-running.